rollup.config.js

```diff
@@ -30,23 +26,23 @@ const createPlugins = ({ isBrowserBuild }) => [
 
 export default [
   {
-    input: "lib/insights.ts",
+    input: "lib/entryBrowser.ts",
```

Comment:
I would have call the file `browser` & `node` but I guess it's fine.

Reply:
Yeah I like that https://github.com/algolia/search-insights.js/pull/140/commits/db1533e10fd380807c551742e6ed04a6c453316a